app/helpers/funding_helper.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 module FundingHelper
+  def training_initiative_options(trainee)
+    TRAINING_ROUTE_INITIATIVES[trainee.training_route]
```

Comment:
Don't we want to use the initiative calculator here?

Reply:
Since it turned out to be just a mapping of `training_route` => `set of initiatives` we decided the simplest thing to do was just store in a hash constant `TRAINING_ROUTE_INITIATIVES`.

If it becomes more involved, we can move into a service if needed.